src/services/pasteEdits.ts

```diff
@@ -167,3 +175,16 @@ function pasteEdits(
         );
     });
 }
+
+function getUsageInfoRangeForPasteEdits({ file: sourceFile, range }: CopiedFromInfo) {
+    // This function adjusts the range so that identifiers at the edges of the copied text are correctly included for imports.
+    // We do not need to adjust range when the tokens at the edges are not identifiers.

```

Comment:
This should be jsdoc. At least, it seems like the comments are intended for callers of the function.